src/public/confirm/styles.css

```
@@ -323,144 +327,362 @@ body {
+
+/* ── Animated error icon ──────────────────────────────────── */
+
+.printer-error-modal__icon-ring {
+  width: 96px;
+  height: 96px;
+  margin-bottom: 24px;
+  animation: error-icon-pop 500ms 200ms cubic-bezier(0.22, 1, 0.36, 1) both;
+}
+
+@keyframes error-icon-pop {
+  from {
```

Stylelint (17.12.0)

[error] Expected "from" to be "0%" (keyframe-selector-notation)

(keyframe-selector-notation)